pipeline_svc/conflict_manager.go

```diff
@@ -448,10 +452,14 @@ func (c *ConflictManager) formatMergedDoc(input *crMeta.ConflictParams, mergedDo
 	}
 
 	bodylen := 0
-	specs := make([]SubdocMutationPathSpec, 0, 5)
+	specs := make([]SubdocMutationPathSpec, 0, 6)
```

Comment:
Same comment.

I also see it a lot of the same reuse for mv path, pv path, etc.
Is there a way assign them to variables and use them?

Adding cvCas requires adding duplicate code in 2 sections with the same spec allocation and spec subdoc spec path.
Any way to avoid code duplication?

Reply:
Done